hw2/03-count.js

```diff
@@ -1,3 +1,35 @@
 // Add your code here
+const input = document.querySelector("input");
+let searchArray = [];
+//const mainTextArray = cleanText();
+let paragraph = document.querySelector("#paragraph");
 
-input.addEventListener('keydown', handleKeyDown);
+let text = document.getElementById("paragraph").innerHTML.toLowerCase();
+input.addEventListener("keydown", handleKeyDown);
+
+function handleKeyDown(e) {
+  if (e.key === "Backspace") {
```

Comment:
I also had struggled with backspace when working on this.  What worked for me was to use the string slice method to remove the last character in the string.  Right now I believe it deletes all the input characters if I only backspace once.

https://byby.dev/js-remove-last-char#:~:text=The%20slice(0%2C%20%2D1,to%20get%20the%20same%20result.